backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/CalendarsService.cs

```diff
@@ -167,10 +172,7 @@ private async Task UpdateVisibleForTeamsTable(Calendar calendar, UserCalendarDto
         {
             _context.CalendarVisibleForTeams.RemoveRange(calendar!.VisibleForTeams);
 
-            foreach (var item in calendar.VisibleForTeams)
-            {
-                await _meetingService.DeleteGoogleCalendarMeetings(item.TeamId);
-            }
+            await RemoveCalendarMeetings(calendar.VisibleForTeams.ToList());
```

Comment:
do we need .ToList here?

Reply:
@tatianahutii yes 
![image](https://user-images.githubusercontent.com/79774156/188332278-8b52f64e-e956-4843-8707-b039f47b0c9b.png)

Reply:
you can change method signature